WTG.Analyzers/Analyzers/ToString/ToStringAnalyzer.cs

```diff
@@ -89,6 +91,13 @@ static void Analyze(SyntaxNodeAnalysisContext context, FileDetailCache cache)
 			}
 		}
 
+		static bool SupportsNameof(SyntaxTree syntaxTree)
+		{
+			var languageVersion = ((CSharpParseOptions)syntaxTree.Options).LanguageVersion;
+
+			return languageVersion == LanguageVersion.Default || languageVersion > LanguageVersion.CSharp5;
```

Comment:
The `== Default` seems odd, but at least we know Roslyn only shipped with C# 6.0 so we can never have an older default.

How would we deal with this for future defaults? For example, VS2019 defaults to 7.3 for most projects and 8.0 for .NET Core 3.0 projects.

Also, why `> CSharp5` and not `>= CSharp6`?

Reply:
Default == 0 the rest are mostly sequential (point releases throw a spanner in the works with C#7.3 == 703 compared to C#8 == 8, but these don't really come into play here.)

I would hope the language version extracted from the syntax tree has `Default` and `Latest` resolved to actual versions, but haven't actually checked.

Technically `>5` and `>=6` are the same but I'll concede that the latter is more semantically correct. I thought about using a switch, but thought that might be more 'icky'.

Reply:
I just checked. If I specify `Default` at the project level, the version extracted from the syntax tree is `CSharp7` ... so it seems we don't need to worry too much about the `Default` value.

Reply:
Oh goody.

Reply:
aha http://source.roslyn.io/#Microsoft.CodeAnalysis.CSharp/CSharpParseOptions.cs,103